### PR TITLE
More tips and some old outdated tip updates

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Adds more tips and rewrites some tips that became outdated due to updates.

## Why It's Good For The Game
More tips is always nice and them having up-to-date information is good too.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>
I got unlucky and didn't see a new/updated tip after launching the game 5 times so no screenshots sadly.
</details>

## Changelog
:cl:
add: Added some new tips.
tweak: Rewrote some outdated tips.
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
+As a Medical Doctor, you can extract implants without breaking them by holding an empty implant case in your offhand while performing the extraction step.
 As a Medical Doctor, clone scanning people will implant them with a health tracker that displays their vitals in the clone records. Useful to check on crew members that didn't activate suit sensors!
 As a Chemist, there are dozens of chemicals that can heal, and even more that can cause harm. Experiment!
 As a Chemist, some chemicals can only be synthesized by heating up the contents with a chemical heater or manually with lighters and similar tools.
-As a Chemist, you will be expected to supply crew with certain chemicals. For example, clonexadone and mannitol for the cryo tubes, unstable mutagen and saltpetre for botany as well as healing pills and patches for the front desk.
+As a Chemist, you will be expected to supply crew with certain chemicals. For example, clonexadone and mannitol for the cryo tubes, plastic for crafting purposes, unstable mutagen and saltpetre for botany as well as healing pills and patches for the front desk.
```
Comment (itsmeow) [inline strings/tips.txt:55]: clonexadone is not a thing, it's cryoxadone

File: strings/tips.txt [hunk 2]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
 As a Chemist, you can make 100u bottles from plastic sheets. The ChemMaster can produce infinite 30u glass bottles as well and if researched the medical techfab can make beakers with capacity ranging from 120u to 300u.
 As a Chemist, you can recharge your chemical dispenser with an inducer or by replacing its cell.
 As a Chemist, Water and Potassium mixed together will create an explosion, with power scaling by amount used. Don't do it.
+As a Chemist, you can create more Bruise Packs and Burn Ointments by using the crafting menu.
+As a Chemist, you can use alternative sources for chemicals other than your chemical dispencer. For example, setting a sheet of paper on fire will create a pile of Ash which you then can scoop up with your beaker. Experiment!
```
Comment (itsmeow) [inline strings/tips.txt]: dispenser

File: strings/tips.txt [hunk 3]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
 As a Geneticist, you can eject someone from cloning early by disabling power in genetics. Note that they will suffer more genetic damage and may lose vital organs from this.
 As a Geneticist, T goes to A, and G goes to C.
 As a Geneticist, becoming a hulk makes you capable of dealing high melee damage, stunlocking people, and punching through walls. However, you can't fire guns, will lose your hulk status if you go into critical condition, and are not considered a human by the AI while you are a hulk.
 As a Geneticist, you have to power to make superhumans. Be aware, going over 99 instability WILL cause side effects.
-As the Virologist, your viruses can range from healing powers so great that you can heal out of critical status, or diseases so dangerous they can kill the entire crew with airborne spontaneous combustion. Experiment!
+As a Geneticist, you can obtain chromosomes by activating mutations. You can then use these chromosomes to ehnance certain mutations.
```
Comment (itsmeow) [inline strings/tips.txt]: enhance. mention how you put the injectors into the console

File: strings/tips.txt [hunk 4]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
+As a Paramedic, it is a good idea to purchase a handheld crew monitor and to keep it open at all times.
```
Comment (itsmeow) [inline strings/tips.txt]: afaik they start with one on their belt

File: strings/tips.txt [hunk 5]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
 As a Medical Doctor, you can surgically implant or extract things from people's chests. This can range from putting in a bomb to pulling out an alien larva.
 As a Medical Doctor, you must target the correct limb and be on help intent when trying to perform surgery on someone. Using disarm attempt will intentionally fail the surgery step.
 As a Medical Doctor, corpses with the "...and their soul has departed" description no longer have a ghost attached to them and aren't revivable or clonable.
-As a Medical Doctor, treating plasmamen is not impossible! Salbutamol stops them from suffocating and showers stop them from burning alive. You can even perform surgery on them by doing the procedure on a roller bed under a shower.
+As a Medical Doctor, treating plasmamen is not impossible! Salbutamol stops them from suffocating and showers stop them from burning alive. You can even perform surgery on them by doing the procedure on a roller bed under a shower or on a statis bed.
```
Comment (Ivniinvi) [inline strings/tips.txt]:
```suggestion
As a Medical Doctor, treating plasmamen is not impossible! Salbutamol stops them from suffocating and showers stop them from burning alive. You can even perform surgery on them by doing the procedure on a roller bed under a shower or on a stasis bed.
```

File: strings/tips.txt [hunk 6]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
 As a Geneticist, T goes to A, and G goes to C.
 As a Geneticist, becoming a hulk makes you capable of dealing high melee damage, stunlocking people, and punching through walls. However, you can't fire guns, will lose your hulk status if you go into critical condition, and are not considered a human by the AI while you are a hulk.
 As a Geneticist, you have to power to make superhumans. Be aware, going over 99 instability WILL cause side effects.
-As the Virologist, your viruses can range from healing powers so great that you can heal out of critical status, or diseases so dangerous they can kill the entire crew with airborne spontaneous combustion. Experiment!
+As a Geneticist, you can obtain chromosomes by activating mutations. You can then use these chromosomes to ehnance certain mutations.
+As a Paramedic, it is a good idea to purchase a handheld crew monitor and to keep it open at all times.
+As a Paramedic, you have access to EVA. Use it to save people who received grievous injuries in space.
+As a Paramedic, you should use other methods of transporting bodies that aren't simply dragging them on the floor : fireman carrying, roller beds, body bags, lockers, etc.
```
Comment (itsmeow) [inline strings/tips.txt]: do mention WHY (blood loss)

File: strings/tips.txt (hunk 5 above)
Comment (CydiaLamiales) [reply strings/tips.txt]: God damnit

File: strings/tips.txt [hunk 7]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
-As a Cyborg, you are immune to most forms of stunning, and excel at almost everything far better than humans. However, flashes can easily stunlock you and you cannot do any precision work as you lack hands.
+As a Cyborg, you are immune to most forms of stunning, and excel at almost everything far better than humans. However, flashes and laser pointers can easily stunlock you and you cannot do any precision work as you lack hands.
 As a Cyborg, you are extremely vulnerable to EMPs as EMPs both stun you and damage you. The ion rifle in the armory or a traitor with an EMP kit can kill you in seconds.
 As a Service Cyborg, your spray can knocks people down. However, it is blocked by masks and glasses.
-As an Engineering Cyborg, you can attach air alarm/fire alarm/APC frames to walls by placing them on the floor and using a screwdriver on them.
+As an Engineering Cyborg, you can attach air alarm/fire alarm/APC/camera frames to walls by placing them on the floor right next to the wall and using a screwdriver on them.
```
Comment (itsmeow) [inline strings/tips.txt:99]: i had no idea this was a thing, holy shit

File: strings/tips.txt [hunk 8]
```diff
@@ -148,24 +165,28 @@ As the Chaplain, you are much more likely to get a response by praying to the go
...
 As a Cargo Technician, you can order contraband items from the supply shuttle console by de-constructing it and using a multitool on the circuit board, the re-assembling it.
 As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, plasma sheets, rare seeds from hydroponics, and more!
+As a Cargo Technician, you can get back some points by stamping the manifest from a crate with the GRANTED rubber stamp, then sending the paper back.
 As the Quartermaster, be sure to check the manifests on crates you receive to make sure all the info is correct. If there's a mistake, stamp the manifest DENIED and send it back in a crate with the items untouched for a refund!
+As the Quartermaster, you can access the Vault by using the supply door remote found in your locker.
 As the Quartermaster, you can construct an express supply console that instantly delivers crates by drop pod. The impact will cause a small explosion as well.
+As the Quartermaster, you can check the Ore Silo logs using your PDA to find out when, where and how many materials were withdrawn or added to it.
```
Comment (itsmeow) [inline strings/tips.txt]: probably mention the Log Viewer app. Captain and HoP also have this

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: strings/tips.txt (hunk 4 above)
Comment (CydiaLamiales) [reply strings/tips.txt]: Suit slot, but yes, they do, forgot about that.

File: strings/tips.txt (hunk 7 above)
Comment (CydiaLamiales) [reply strings/tips.txt:99]: The problem is that your sprite obscures it, need to alt-click the tile under yourself.

File: strings/tips.txt [hunk 9]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
 As a Geneticist, you can eject someone from cloning early by disabling power in genetics. Note that they will suffer more genetic damage and may lose vital organs from this.
 As a Geneticist, T goes to A, and G goes to C.
 As a Geneticist, becoming a hulk makes you capable of dealing high melee damage, stunlocking people, and punching through walls. However, you can't fire guns, will lose your hulk status if you go into critical condition, and are not considered a human by the AI while you are a hulk.
 As a Geneticist, you have to power to make superhumans. Be aware, going over 99 instability WILL cause side effects.
-As the Virologist, your viruses can range from healing powers so great that you can heal out of critical status, or diseases so dangerous they can kill the entire crew with airborne spontaneous combustion. Experiment!
+As a Geneticist, you can obtain chromosomes by activating mutations by recycling used activators through your console or activating mutatins with the console. You can then use these chromosomes to ehnance certain mutations.
```
Comment (itsmeow) [inline strings/tips.txt]: "mutatins" "ehnance"

File: strings/tips.txt [hunk 10]
```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
...
+As a Paramedic, it is a good idea to keep your handheld crew monitor's inferface open at all times.
```
Comment (itsmeow) [inline strings/tips.txt]: inferface

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Review by itsmeow — APPROVED** [review]

**Pull request merged**